plugins/tutor-contrib-paragon/README.rst

```diff
@@ -60,9 +59,107 @@ All configuration variables can be overridden via `tutor config save`:
       - theme-1
       - theme-2
     PARAGON_SERVE_COMPILED_THEMES: true
+    PARAGON_BUILDER_IMAGE: "paragon-builder:latest"
 
 You may customize paths or theme names to suit your deployment.
 
+Usage
+*****
+
+Prerequisites
+-------------
+
+- A built Paragon CLI image:
+
+  .. code-block:: bash
+
+      tutor images build paragon-builder
+
+- A theme source directory structured as follows:
```

Comment:
It'd be nice to clarify that `theme-sources` in this tree is referring to the directory specified in  `PARAGON_THEME_SOURCES_PATH` here.